test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -74,3 +74,19 @@ def test_path_from_cli_depend_on_cwd(base_arguments, monkeypatch):  # Issue 572
 
     assert 'test/test' not in options1.exclude_paths[0]
     assert 'test/test' in options2.exclude_paths[0]
+
+
+def test_invalid_config_file(base_arguments):
+    """Ensures invalid config_file produces error code 2."""
+    with pytest.raises(SystemExit) as excinfo:
+        cli.get_config(base_arguments +
+                       ["-c", "test/fixtures/ansible-config-invalid.yml"])
+    assert excinfo.value.code == 2
+
+
+def test_missing_config_file(base_arguments):
+    """Ensures missing config_file produces error code 2."""
+    with pytest.raises(SystemExit) as excinfo:
+        cli.get_config(base_arguments +
+                       ["-c", "test/fixtures/ansible-config-missing.yml"])
```

Comment:
What if a file with this name appears there?
```suggestion
                       ["-c", "/dev/null/ansible-config-missing.yml"])
```